x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -143,34 +350,22 @@ export class AlertsClient {
   }: GetAlertParams): Promise<ParsedTechnicalFields | null | undefined> {
     try {
       // first search for the alert by id, then use the alert info to check if user has access to it
-      const alert = await this.fetchAlert({
+      const alert = await this.fetchAlertAndAudit({
         id,
+        query: null,
         index,
-      });
-
-      if (alert == null) {
-        return;
-      }
-
-      // this.authorization leverages the alerting plugin's authorization
-      // client exposed to us for reuse
-      await this.authorization.ensureAuthorized({
-        ruleTypeId: alert[RULE_ID],
-        consumer: alert[OWNER],
         operation: ReadOperations.Get,
-        entity: AlertingAuthorizationEntity.Alert,
+        lastSortIds: undefined,
       });
 
-      this.auditLogger?.log(
-        alertAuditEvent({
-          action: AlertAuditAction.GET,
-          id,
-        })
-      );
+      if (alert == null || alert.hits.hits.length === 0) {
+        const errorMessage = `Unable to retrieve alert details for alert with id of "${id}" and operation ${ReadOperations.Get}`;
+        this.logger.error(errorMessage);
+        throw Boom.notFound(errorMessage);
+      }
 
-      return alert;
+      return alert.hits.hits[0]._source;
```

Comment:
Should we add a to do to remind us that we have to follow up to remove us off of `_source`?